Re-import css-view-transitions WPTs.

mattwoodrow opened the pull request:
#### b4239a38c109a1f5980fc9c1cfe9a98cd3328741
<pre>
Re-import css-view-transitions WPTs.
<a href="https://bugs.webkit.org/show_bug.cgi?id=281191">https://bugs.webkit.org/show_bug.cgi?id=281191</a>
&lt;<a href="https://rdar.apple.com/137644004">rdar://137644004</a>&gt;

Reviewed by Tim Nguyen.

Upstream commit: <a href="https://github.com/web-platform-tests/wpt/commit/0ca2856475e2d067f7f2523ffb30de965c28d14a">https://github.com/web-platform-tests/wpt/commit/0ca2856475e2d067f7f2523ffb30de965c28d14a</a>

* LayoutTests/imported/w3c/resources/resource-files.json:
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/auto-name-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/auto-name-from-id-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/auto-name-from-id-shadow-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/auto-name-from-id-shadow.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/auto-name-from-id.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/auto-name-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/auto-name.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/element-is-grouping-during-animation.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/navigation/auto-name-from-id-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/navigation/auto-name-from-id-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/navigation/auto-name-from-id.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/navigation/custom-scrollbar-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/navigation/custom-scrollbar-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/navigation/custom-scrollbar.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/navigation/resources/auto-name-from-id.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/navigation/resources/custom-scrollbar.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/navigation/resources/w3c-import.log:
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/navigation/root-element-transition-iframe-with-startVT-on-main.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/navigation/w3c-import.log:
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/new-and-old-sizes-match.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/new-content-captures-different-size.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/old-content-captures-different-size.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/parsing/pseudo-elements-invalid.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/parsing/view-transition-name-invalid.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/parsing/view-transition-name-valid.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/transform-origin-view-transition-group.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/w3c-import.log:

Canonical link: <a href="https://commits.webkit.org/285006@main">https://commits.webkit.org/285006@main</a>
</pre>
<!--EWS-Status-Bubble-Start-->
https://github.com/WebKit/WebKit/commit/3cf264491b0644811233cf97aab9df27af73bbda

| Misc | iOS, visionOS, tvOS & watchOS  | macOS  | Linux |  Windows |
| ----- | ---------------------- | ------- |  ----- |  --------- |
| [✅ 🧪 style](https://ews-build.webkit.org/#/builders/38/builds/71190 "Passed style check") | [✅ 🛠 ios](https://ews-build.webkit.org/#/builders/48/builds/50603 "Built successfully") | [✅ 🛠 mac](https://ews-build.webkit.org/#/builders/55/builds/23963 "Built successfully") | [✅ 🛠 wpe](https://ews-build.webkit.org/#/builders/5/builds/75297 "Built successfully") | [✅ 🛠 win](https://ews-build.webkit.org/#/builders/59/builds/22395 "Built successfully") 
| | [![loading](https://user-images.githubusercontent.com/3098702/171232313-daa606f1-8fd6-4b0f-a20b-2cb93c43d19b.png) 🛠 ios-sim](https://ews-build.webkit.org/#/builders/49/builds/58403 "Build is in progress. Recent messages:OS: Sonoma (14.5), Xcode: 15.4; Running apply-patch; Checked out pull request; Compiled WebKit (warnings)") | [✅ 🛠 mac-AS-debug](https://ews-build.webkit.org/#/builders/61/builds/22214 "Built successfully") | [✅ 🧪 wpe-wk2](https://ews-build.webkit.org/#/builders/34/builds/56276 "Passed tests") | [✅ 🧪 win-tests](https://ews-build.webkit.org/#/builders/60/builds/14738 "Passed tests") 
| [✅ 🧪 webkitperl](https://ews-build.webkit.org/#/builders/11/builds/74256 "Passed tests") | [✅ 🧪 ios-wk2](https://ews-build.webkit.org/#/builders/47/builds/45977 "Passed tests") | [✅ 🧪 api-mac](https://ews-build.webkit.org/#/builders/18/builds/61353 "Passed tests") | [✅ 🧪 api-wpe](https://ews-build.webkit.org/#/builders/41/builds/36712 "Passed tests") | 
| [⏳ 🧪 webkitpy ](https://ews-build.webkit.org/#/builders/WebKitPy-Tests-EWS "Waiting in queue, processing has not started yet") | [✅ 🧪 ios-wk2-wpt](https://ews-build.webkit.org/#/builders/42/builds/42650 "Passed tests") | [✅ 🧪 mac-wk1](https://ews-build.webkit.org/#/builders/64/builds/18815 "Passed tests") | [✅ 🛠 wpe-cairo](https://ews-build.webkit.org/#/builders/65/builds/20735 "Built successfully") | 
| | [✅ 🧪 api-ios](https://ews-build.webkit.org/#/builders/13/builds/64542 "Passed tests") | [✅ 🧪 mac-wk2](https://ews-build.webkit.org/#/builders/63/builds/19178 "Passed tests") | [✅ 🛠 gtk](https://ews-build.webkit.org/#/builders/2/builds/77017 "Built successfully") | 
| | [✅ 🛠 vision](https://ews-build.webkit.org/#/builders/87/builds/15424 "Built successfully") | [![loading](https://user-images.githubusercontent.com/3098702/171232313-daa606f1-8fd6-4b0f-a20b-2cb93c43d19b.png) 🧪 mac-AS-debug-wk2](https://ews-build.webkit.org/#/builders/62/builds/18359 "Build is in progress. Recent messages:OS: Sonoma (14.5), Xcode: 15.4; Running apply-patch; Checked out pull request; Passed layout tests; 17 flakes") | [✅ 🧪 gtk-wk2](https://ews-build.webkit.org/#/builders/1/builds/63994 "Passed tests") | 
| | [![loading](https://user-images.githubusercontent.com/3098702/171232313-daa606f1-8fd6-4b0f-a20b-2cb93c43d19b.png) 🛠 vision-sim](https://ews-build.webkit.org/#/builders/86/builds/15466 "Build is in progress. Recent messages:OS: Sonoma (14.5), Xcode: 15.4; Running apply-patch; Checked out pull request; Compiled WebKit (warnings)") | [✅ 🧪 mac-wk2-stress](https://ews-build.webkit.org/#/builders/8/builds/61388 "Passed tests") | [✅ 🧪 api-gtk](https://ews-build.webkit.org/#/builders/21/builds/63970 "Passed tests") | 
| [✅ 🛠 🧪 merge](https://ews-build.webkit.org/#/builders/19/builds/15761 "Built successfully and passed tests") | [✅ 🧪 vision-wk2](https://ews-build.webkit.org/#/builders/88/builds/12113 "Passed tests") | [✅ 🧪 mac-intel-wk2](https://ews-build.webkit.org/#/builders/119/builds/5743 "Passed tests") | | 
| | [✅ 🛠 tv](https://ews-build.webkit.org/#/builders/44/builds/46403 "Built successfully") | | | 
| | [✅ 🛠 tv-sim](https://ews-build.webkit.org/#/builders/46/builds/47474 "Built successfully") | | | 
| | [✅ 🛠 watch](https://ews-build.webkit.org/#/builders/43/builds/48757 "Built successfully") | | | 
| | [✅ 🛠 watch-sim](https://ews-build.webkit.org/#/builders/45/builds/47216 "Built successfully") | | | 
<!--EWS-Status-Bubble-End-->

nt1m commented on LayoutTests/platform/mac/imported/w3c/web-platform-tests/css/css-view-transitions/layered-capture/tree-effects.tentative.sub.html?prop=mask-image&old=linear-gradient(rgb(0%200%200%20/%2080%),%20transparent)&new=none-expected.txt:
can we delete this file?

mattwoodrow committed:
Re-import css-view-transitions WPTs.
https://bugs.webkit.org/show_bug.cgi?id=281191
<rdar://137644004>

Reviewed by Tim Nguyen.

Upstream commit: https://github.com/web-platform-tests/wpt/commit/0ca2856475e2d067f7f2523ffb30de965c28d14a

* LayoutTests/imported/w3c/resources/resource-files.json:
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/auto-name-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/auto-name-from-id-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/auto-name-from-id-shadow-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/auto-name-from-id-shadow.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/auto-name-from-id.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/auto-name-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/auto-name.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/element-is-grouping-during-animation.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/navigation/auto-name-from-id-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/navigation/auto-name-from-id-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/navigation/auto-name-from-id.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/navigation/custom-scrollbar-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/navigation/custom-scrollbar-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/navigation/custom-scrollbar.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/navigation/resources/auto-name-from-id.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/navigation/resources/custom-scrollbar.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/navigation/resources/w3c-import.log:
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/navigation/root-element-transition-iframe-with-startVT-on-main.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/navigation/w3c-import.log:
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/new-and-old-sizes-match.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/new-content-captures-different-size.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/old-content-captures-different-size.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/parsing/pseudo-elements-invalid.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/parsing/view-transition-name-invalid.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/parsing/view-transition-name-valid.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/transform-origin-view-transition-group.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/w3c-import.log:

Canonical link: https://commits.webkit.org/285006@main
